finality-aleph/src/network/tcp.rs

```diff
@@ -43,7 +43,7 @@ impl SecretKey for AuthorityPen {
     type PublicKey = AuthorityIdWrapper;
 
     async fn sign(&self, message: &[u8]) -> Self::Signature {
-        AuthorityPen::sign(self, message).await
+        AuthorityPen::sign(self, message)
```

Comment:
So we can make `SecretKey` sync now, right? It would be nice to do that in this PR, but if it turns out to be too much work to propagate all the changes then at least create a task.

Reply:
yep, done